addon/styles/_list-item-primitives.scss

```diff
@@ -0,0 +1,185 @@
+@import 'node_modules/ember-frost-core/addon/styles/frost-theme';
+
+$frost-list-item-primary-state-critical: #E52C2C;
+$frost-list-item-primary-state-major: #EF8E00;
+$frost-list-item-primary-state-warn: #7D85B2;
+
+.frost-list-item-primary-identifier {
+  overflow: hidden;
+  white-space: nowrap;
+  text-overflow: ellipsis;
+  font-size: $frost-font-s;
+  color: $frost-color-grey-1;
+  font-weight: 600;
+  line-height: 30px;
+}
+
+.frost-list-item-secondary-identifier {
+  overflow: hidden;
+  white-space: nowrap;
+  text-overflow: ellipsis;
+  font-size: $frost-font-xs;
+  color: $frost-color-grey-1;
+  line-height: 30px;
+}
+
+.frost-list-item-unlabeled-property {
+  overflow: hidden;
+  white-space: nowrap;
+  text-overflow: ellipsis;
+  font-size: $frost-font-xs;
+  color: $frost-color-grey-1;
+  line-height: 30px;
+}
+
+.frost-list-item-boolean {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  height: 30px;
+  .frost-list-item-boolean-label {
+    font-size: $frost-font-xs;
+    font-weight: 700;
+    color: $frost-color-grey-4;
+    margin-right: 5px;
+    min-width: 0;
+  }
+  .frost-list-item-boolean-icon {
+    position: relative;
+    top: 2px;
+    svg {
+      height: 15px;
+      width: 15px;
+    }
+  }
+}
+
+.frost-list-item-large-icon-text {
+  display: flex;
+  flex-direction: row;
+  height: 60px;
+  align-items: center;
+
+  .frost-list-item-large-icon-text-text {
+    font-size: $frost-font-xs;
+    color: $frost-color-grey-1;
+    min-width: 0;
+  }
+
+  .frost-list-item-large-icon-text-icon {
+    position: relative;
+    top: 2px;
+    margin-right: 10px;
+
+    svg {
+      height: 40px;
+      width: 40px;
+    }
+  }
+}
+
+.frost-list-item-icon-text {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  height: 30px;
+
+  .frost-list-item-icon-text-text {
+    min-width: 0;
+    font-size: $frost-font-xs;
+    color: $frost-color-grey-4;
+  }
+  .frost-list-item-icon-text-icon {
+    position: relative;
+    top: 2px;
+    margin-right: 5px;
+    svg {
+      height: 20px;
+      width: 20px;
+    }
+  }
+}
+
+.frost-list-item-small-view {
```

Comment:
Ahhh, now I get where the component was coming from before...this is interesting.  If we talk about the structure of the "header"/"small" view of a list item then there are definitely conventions that could be put in place, but it would become more restrictive I believe - let's chat about this with a group - maybe we can schedule 30 minutes or so with the same group reviewing the object browser?